### PR TITLE
STATUS11 BatteryUpdate

### PR DESCRIPTION
**Pull request by stefanbode**
Added BatteryPercentage announcement to Status11. Supported in current TASMOTA versions

### CODE REVIEW

**Review by emontnemery — DISMISSED** [review]
Thanks, @stefanbode 👍

File: hatasmota/device_status.py
Comment (emontnemery) [inline hatasmota/device_status.py:1]: Why do we add battery percentage here?

**Comment by stefanbode** [comment]
Status11 is identically with the STATUS message. Therefore I did not need to make any changes in TASMOTA. It just show up as default. It also makes totally sense here because this is a Information about current device information and the Battery makes sense here.

File: hatasmota/device_status.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 from .const import (
     CONF_MAC,
+    SENSOR_STATUS_BATTERY_PERCENTAGE,
```
Comment (emontnemery) [inline hatasmota/device_status.py:11]:
```suggestion
```

File: hatasmota/device_status.py [hunk 2]
```diff
@@ -37,7 +38,7 @@
 #  "Module or Template":"Generic",             <NONE>
 #  "RestartReason":"Software/System restart",  stat/STATUS1:"StatusPRM"."RestartReason"
 #  "Uptime":"1T17:04:28",                      stat/STATUS11:"StatusSTS"."Uptime"; tele/STATE:"Uptime"
-#  "BatteryPercentage":60,                     tele/STATE: "BatteryPercentage"
+#  "BatteryPercentage":60,                     stat/STATUS11:"StatusSTS"."BatteryPercentage"; tele/STATE: "BatteryPercentage"
```
Comment (emontnemery) [inline hatasmota/device_status.py:41]:
```suggestion
```

File: hatasmota/device_status.py [hunk 3]
```diff
@@ -63,6 +64,7 @@
     SENSOR_STATUS_RSSI: ["StatusSTS", "Wifi", "RSSI"],
     SENSOR_STATUS_SIGNAL: ["StatusSTS", "Wifi", "Signal"],
     SENSOR_STATUS_SSID: ["StatusSTS", "Wifi", "SSId"],
+    SENSOR_STATUS_BATTERY_PERCENTAGE: ["StatusSTS", "BatteryPercentage"],
```
Comment (emontnemery) [inline hatasmota/device_status.py:67]:
```suggestion
```

**Review by emontnemery — CHANGES_REQUESTED** [review]
We can remove the changes in hatasmota/device_status.py, that feature is not used by Home Assistant

**Review by emontnemery — APPROVED** [review]

**Pull request merged**